bin/ch-fromhost

```diff
@@ -228,29 +294,11 @@ done
 
 [ "$image" ] || fatal "no image specified"
 
-if [ $cray_mpi ]; then
-    sentinel=/etc/opt/cray/release/cle-release
-    [ -f $sentinel ] || fatal "not found: ${sentinel}: are you on a Cray?"
-
-    mpi_version=$("${ch_bin}/ch-run" "$image" -- mpirun --version || true)
-    case $mpi_version in
-        *mpich*)
-            cray_mpich=yes
-            ;;
-        *'Open MPI'*)
-            cray_openmpi=yes
-            ;;
-        *)
-            fatal "can't find MPI in image"
-            ;;
-    esac
-fi
-
 if [ $lib_found ]; then
     # We want to put the libraries in the first directory that ldconfig
     # searches, so that we can override (or overwrite) any of the same library
     # that may already be in the image.
-    debug "asking ldconfig for shared library destination"
+    debug "[ debug ] asking ldconfig for shared library destination"
```

Comment:
`[ debug ]` redundant?

Reply:
The intention was to distinguish debug output from regular output.

```
  $ ch-fromhost --ofi $HOME/scratch/opt/lib/libfabric/libginx-fi.so /var/tmp/openmpi
  [ debug ]   found /home/cholo/scratch/opt/lib/libfabric/libgnix-fi.so
  [ debug ] searching /var/tmp/openmpi for libfabric dso provider destination...
  [ debug ]   found: /var/tmp/openmpi//usr/local/lib/libfabric.so
[...]
```

However it looks like the debug output is always printed, regardless of whether `-v` is passed or not (even for other `--path` and `--file` options.

Reply:
Right; but on line 301 it will be printed twice because function `debug` already adds it. Unless I'm missing something?